### PR TITLE
FileVersion fix

### PR DESCRIPTION
**Pull request by Green7**

### CODE REVIEW

**Review by dgolub — COMMENTED** [review]
Is this really a bug?  If there are no translations in the file, then an empty array seems like a more accurate representation than returning an error.

**Comment by Green7** [comment]
Yes it is. I have bug reporting in my application and many users get the error:
**runtime error: index out of range [0] with length 0**
in fileversion.go in (*FileVersion).GetFirstStringFileInfo at line 289

The question is whether to fix it in GetTranslations() or in .GetFirstStringFileInfo().

**Comment by dgolub** [comment]
Got it.  You're right; there is a bug.  We should change `GetFirstStringFileInfo` to check that the size of the array is nonzero.

**Comment by Green7** [comment]
@dgolub  Can you look at it ?

File: fileversion.go [hunk 1]
```diff
@@ -286,6 +287,9 @@ func (self *FileVersion) GetFirstStringFileInfo() (*StringFileInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	if len(tr) == 0 {
+		return nil, errors.New("Translations is empty");
```
Comment (dgolub) [inline fileversion.go:291]: [Nitpick] Don't need the semicolon here.  Also, the Go convention is for error messages to start with a lowercase letter.  Something like `"no translations"` would be good.

**Review by dgolub — COMMENTED** [review]
Sorry about that--I never got a notification that this was ready for review again.

**Review by dgolub — APPROVED** [review]
👍

File: fileversion.go (hunk 1 above)
Comment (Green7) [reply fileversion.go]: You right, corrected.

**Pull request merged**